scripts/build_ios.sh

```diff
@@ -88,6 +88,12 @@ if [ "${TRACING_BASED}" == 1 ]; then
 else
   CMAKE_ARGS+=("-DTRACING_BASED=OFF")
 fi
+if [ "${USE_LIGHTWEIGHT_DISPATCH}" == 1 ]; then
+  CMAKE_ARGS+=("-DUSE_LIGHTWEIGHT_DISPATCH=ON")
+  CMAKE_ARGS+=("-DSTATIC_DISPATCH_BACKEND=CPU")
```

Comment:
I'm wondering if there's actual use case with STATIC_DISPATCH_BACKEND other than CPU? Even with USE_LIGHTWEIGHT_DISPATCH on, there could be multiple backends that need to be code-gen, like CPU, QuantizedCPU. Any reason there are two switches of USE_LIGHTWEIGHT_DISPATCH and TATIC_DISPATCH_BACKEND?

Reply:
So `USE_LIGHTWEIGHT_DISPATCH` not only generate kernels with static dispatch backend(s), but also generate C++ code to register them into JIT op registry. I think it makes sense to keep them separated.

> there could be multiple backends that need to be code-gen, like CPU, QuantizedCPU
Yep we will support multiple backends for static dispatch as next step.

I guess it makes sense to not bundle `USE_LIGHTWEIGHT_DISPATCH` and `STATIC_DISPATCH_BACKEND` together. I put it this way just for testing purpose, will change when I publish it.

Reply:
I see, so 

USE_LIGHTWEIGHT_DISPATCH:  STATIC_DISPATCH_BACKEND must be set + JIT op registry.

STATIC_DISPATCH_BACKEND: specify which backend to statically dispatch to. In future, it may be extended to support multiple backends.

It would be great to have the logic in the PR summary and also in the comments.